package/yast2-registration.spec

```diff
@@ -58,6 +58,13 @@ BuildRequires:  yast2-update >= 3.1.19
 
 BuildArch:      noarch
 
+# FIXME: it seems can we cannot move it to macros.yast, the yast-rake-ci is not
+# installed into the chroot, the build fails...
```

Comment:
Ah, you mean moving this BuildRequires somewhere else?

What about declaring `%bcond_and_buildrequires_with_yast_run_ci_tests` in yast.macros, which expands to the lines below?